app/models/post.rb

```diff
@@ -1,2 +1,17 @@
 class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  after_save :update_post_counter
+
+  private
+
+  def update_post_counter
+    posts.increment!(:posts_counter)
```

Comment:
[REQUIRED] Your method should increment the counter for a specific user:
`author.increment!(:posts_counter)`